projects/extension/vite.script.config.js

```diff
@@ -14,12 +13,7 @@ export default defineConfig(({ mode }) => ({
       },
       output: {
         entryFileNames: (chunk) => `${chunk.name}.js`,
-        inlineDynamicImports: !isBackground,
-        manualChunks: isBackground
-          ? {
-              "substrate-connect": ["@substrate/connect"],
-            }
-          : undefined,
```

Comment:
The background script size is 2.8Mb (< 4Mb JS file size limit required for any extension script), so there is no need to do `manualChunks`